test/go.mod

```diff
@@ -5,10 +5,10 @@ go 1.25.0
 require (
 	github.com/btcsuite/btcd/chaincfg/chainhash v1.0.2
 	github.com/onflow/atree v0.11.0
-	github.com/onflow/cadence v1.8.3
-	github.com/onflow/flow-emulator v1.10.4
-	github.com/onflow/flow-go v0.43.3-0.20251029192457-d4f875155987
-	github.com/onflow/flow-go-sdk v1.9.2
+	github.com/onflow/cadence v1.8.3-0.20251104182324-d1e559944703
```

Comment:
This should be updated to the new Cadence release after https://github.com/onflow/cadence/pull/4331 got merged